src/main/java/gregtech/api/capability/impl/AbstractRecipeLogic.java

```diff
@@ -54,6 +54,8 @@
     protected boolean workingEnabled = true;
     protected boolean hasNotEnoughEnergy;
     protected boolean wasActiveAndNeedsUpdate;
+    protected boolean isOutputsFull;
```

Comment:
Would there ever be a case where we want to differentiate between if the Fluid Outputs are full and if the item outputs are full? Maybe for something like the Electrolyzer?

Reply:
the same effect would be achieved by testing for ex.: ( isItemOutpuFull || isFluidOutputFull ) for recipe purposes.
setupAndConsumeRecipeInputs requires both to merge successfully before returning true